src/modules/dashboard/components/ColonyHomeActions/ColonyHomeActions.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Button from '~core/Button';
+import ColonyActionsDialog from './ColonyActionsDialog';
+import { useDialog } from '~core/Dialog';
+
+const displayName = 'dashboard.ColonyHomeActions';
```

Comment:
This should have a more descriptive name. Something like: `ColonyHomeCreateActionsButton`